gdpr_assist/models.py

```diff
@@ -246,33 +278,278 @@ def _cast_class(cls, model, privacy_meta):
 
         return model
 
+    def get_anon_log(self):
```

Comment:
Keep the naming consistent?

```suggestion
    def get_anonymisation_log(self):
```